src/gui/guiEngine.cpp

```diff
@@ -297,10 +297,14 @@ void GUIEngine::run()
 
 		driver->endScene();
 
+		IrrlichtDevice *device = RenderingEngine::get_raw_device();
+		u32 frametime_min = 1000 / (device->isWindowFocused()
+			? g_settings->getFloat("fps_max")
+			: g_settings->getFloat("fps_max_unfocused"));
```

Comment:
It would be great if `Game::limitFps` could become a global class to handle the framerate the same in all RenderingEngine loops.